test/unit/com/unifina/service/StreamServiceSpec.groovy

```diff
@@ -382,6 +380,7 @@ class StreamServiceSpec extends Specification {
 		1 * service.permissionService.getPermissionsTo(stream, Permission.Operation.STREAM_PUBLISH) >> perms
 		addresses == validAddresses
 	}
+*/
```

Comment:
- The test `getStreamEthereumPublishers should return Ethereum addresses of users with write permission to the Stream` should not be removed
- Remove obsolete tests instead of commenting them out